blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_7_serviceability_limit_state.py

```diff
@@ -0,0 +1,49 @@
+"""Module including all formulas from chapter 7 of NEN-EN 1992-1-1+C2:2011."""
+# pylint: disable=arguments-differ
+from blueprints.codes.formula import Formula
+from blueprints.typehints import KN, KN_TO_N, MM2, MPA
+
+
+class Form7Dot3CoefficientKc(Formula):
+    """Class representing the formula 7.3 for the coefficient kc for flanges of tubular cross-sections and T-sections [-]."""
+
+    label = "7.3"
+    source_document = "NEN-EN 1992-1-1+C2:2011"
```

Comment:
could be imported from the init of the NEN-EN 1992-1-1+C2:2011 package. Where it is defined as a constant

Reply:
I prefer to keep things simple, but I'm open to initializing it in the document module. What do others think?

Reply:
Importing a string a string from a parent folder isn't that complex isn't it? :-p I think it prevents mistakes in the following formulas.